### ISSUE TITLE
[Bug] Broken pipe and WebSocket error

### ISSUE THREAD

**Issue by 5frank** [thingsboard/thingsboard#4078]
**Describe the bug**
Broken pipe and WebSocket error.

**Your Server Environment**

* own setup
*  docker
  * ThingsBoard Version: 3.2.0PE
  * OS Name and Version: Ubuntu 20.04.1 LTS

**Your Client Environment**

N/A this is probably  a server side issue.

I get the following error from Thingsboard server `docker logs`:

```
[http-nio-0.0.0.0-8080-exec-3] ERROR o.a.c.c.C.[Tomcat].[localhost] - Exception Processing ErrorPage[errorCode=0, location=/error]
org.apache.catalina.connector.ClientAbortException: java.io.IOException: Broken pipe
[... long traceback ...]
Caused by: java.io.IOException: Broken pipe
	at sun.nio.ch.FileDispatcherImpl.write0(Native Method)

```

It seems to be related to a failing web socket? 
Widget data is not updated and I get the following error from the browser debugger:

`WebSocket error event`  on url "wss://[our tb server]/api/ws/plugins/telemetry?token=[long token]"

Any suggestion? Is it a permission issue? The Thingsboard  log is very verbose but I fail to find the cause.

**Comment by pon0marev** [comment]
If possible, upgrade ThingsBoard to a newer version. Many improvements and fixes have been implemented since version 3.2.0, including web socket performance optimizations.
If you encounter this issue again after upgrading, check your load balancer settings. You may need to increase the timeout at the load balancer level.

**Comment by pon0marev** [comment]
We want to inform you that we have closed your ticket as we have not received any updates on this issue. If you still have questions or encounter similar issues after upgrading to the latest version, please reopen this issue.
